src/components/RedigerAvansertBeregning/RedigerAvansertBeregning.tsx

```diff
@@ -409,6 +327,24 @@ export const RedigerAvansertBeregning: React.FC<{
       : ''
   }
 
+  // TODO flytte denne til en util og bruk memo med array: tidligstMuligHeltUttak, localGradertUttak
+  const getAgePickerHelDescription = (): string | undefined => {
```

Comment:
Lurer litt på hva "hel description" er for noe.

Reply:
"hel" var ment til å beskrivelse hel-uttaksalder for AgePicker, ikke beskrivelsen. Men ser at det ikke er veldig lesbart.
Kanskje heller `getHeltUttakAgePickerBeskrivelse`? eller andre forslag?

Reply:
Synes `getHeltUttakAgePickerBeskrivelse` er bra :)